mipengine/tests/controller/api/test_get_algorithms.py

```diff
@@ -0,0 +1,178 @@
+import json
+import logging
+import unittest
```

Comment:
Let's use pytest instead of unittest.

Reply:
Removed it completely with the unittest.main() as well.

Was needlessly copied from the exareme tests.

pytest is being used to run the tests.